### PR TITLE
(#546) Clarify when runtime virus scanning is performed

### LINKED ISSUE #546: Add Better Wording that Virus Check Only Triggers on Non-Internalized Package Files
## What New Or Updated Would You Like To See?

Clearly document when/how virus scanning works when using both embedded packages and packages that reach out an external source for assets.

## Why Is It Needed?

There is currently confusion around when virus scanning actually takes place when you enable the features in a licensed Chocolatey installation.

## Additional Context?

NA

## References

 - [ZenDesk Ticket](https://chocolatey.zendesk.com/agent/tickets/15142)

### PR DESCRIPTION
**Pull request by vexx32**
## Description Of Changes

- Clarify that runtime virus scanning does not submit embedded binaries to the virus scanner and only operates on files downloaded for the installation.

## Motivation and Context

See #546

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [x] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.
* [ ] The change I have made should have a video added, and I have raised an issue for this.
    * Issue #

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue

Fixes #546

### CODE REVIEW

File: input/en-us/choco/release-notes.md [hunk 1]
```diff
@@ -36,6 +36,10 @@ This covers changes for the "chocolatey" and "chocolatey.lib" packages, which ar
 
 - Allow post/pre hooks to run on upgrade/install/uninstall - see [#1185](https://github.com/chocolatey/choco/issues/1185)
 
+### Deprecated Features
```
Comment (pauby) [inline input/en-us/choco/release-notes.md:39]: Is this change supposed to be here?

File: input/en-us/features/virus-check.md [hunk 2]
```diff
@@ -6,7 +6,11 @@ Description: Virus checking via VirusTotal or Generic scanner
 RedirectFrom: docs/features-virus-check
 ---
 
-We are pleased to offer runtime malware protection for our [customers](https://chocolatey.org/pricing)! Virus checking is available in two flavors - VirusTotal and Generic malware verification.
+We are pleased to offer runtime malware protection for our [customers](https://chocolatey.org/pricing)!
+Virus checking is available in two flavors - VirusTotal and Generic malware verification.
+
+Both forms of virus scanning will only scan files **downloaded** from an external source during the package installation process.
+Packages which embed the software binaries will not have their binaries scanned during installation.
```
Comment (pauby) [inline input/en-us/features/virus-check.md:13]: Could we clarify that they won't have the binaries scanned by Chocolatey. Their AV (if they have one) may do it, though. But the way it is written, we are saying they won't be scanned at all.

**Review by pauby — CHANGES_REQUESTED** [review]

File: input/en-us/choco/release-notes.md (hunk 1 above)
Comment (vexx32) [reply input/en-us/choco/release-notes.md:39]: Huh, no. I'll see why that got pulled in.... O.o

File: input/en-us/features/virus-check.md (hunk 2 above)
Comment (vexx32) [reply input/en-us/features/virus-check.md:13]: Yep, that sounds good, will do.

Comment (vexx32) [reply input/en-us/features/virus-check.md]: Fixed this up. :)

File: input/en-us/choco/release-notes.md (hunk 1 above)
Comment (vexx32) [reply input/en-us/choco/release-notes.md]: Dropped this wayward commit

**Review by pauby — APPROVED** [review]

**Pull request merged**